requirements/requirements.txt

```diff
@@ -71,6 +71,7 @@ pip>=6.1.1
 git+git://github.com/smartfile/django-transfer.git@6e0dc94c3341c358fca8eb2bf74e23aee3983ec4
 Pygments==2.0.2
 tinys3==0.1.11
+django-two-factor-auth==1.2.0
```

Comment:
Would be nice to bump this to 1.2.2 as that uses django-otp 0.3 which supports django migrations (https://github.com/Bouke/django-two-factor-auth/compare/1.2.0...releases/1.2.2 & https://pythonhosted.org/django-otp/changes.html#v0-3-0-february-7-2015-support-django-migrations)

Would be nice because if we want to upgrade this library we don't need to worry about the upgrade path later.